runtime/near-vm-logic/src/types.rs

```diff
@@ -29,6 +29,7 @@ pub enum ReturnData {
 /// calls are available to the contract invoked through the callback.
 #[derive(Debug, PartialEq, Serialize, Deserialize)]
 pub enum PromiseResult {
+    /// [deprecated] NotReady is not used any more, it comes from the early versions of the protocol.
```

Comment:
```suggestion
    /// Current version of the protocol never returns `PromiseResult::NotReady`.
```

It's not really deprecated. More like `unimplemented`.

Reply:
Good suggestion. Thanks.